msmbuilder/tests/test_commands.py

```diff
@@ -149,7 +149,7 @@ def test_superpose_featurizer_reftop():
         shell("msmb SuperposeFeaturizer "
               "--trjs '{data_home}/alanine_dipeptide/*.dcd'"
               " --transformed distances --atom_indices all.txt"
-              " --reference_traj {data_home}/alanine_dipeptide/trajectory_0.dcd"
+              " --reference_traj {data_home}/alanine_dipeptide/trajectory-0.dcd"
```

Comment:
I think figshare changed the naming scheme somehow, replacing underscores with hyphens.